alicloud/resource_alicloud_kvstore_instance.go

```diff
@@ -368,6 +367,53 @@ func resourceAlicloudKVStoreInstanceUpdate(d *schema.ResourceData, meta interfac
 		return resourceAlicloudKVStoreInstanceRead(d, meta)
 	}
 
+	if d.HasChange("vswitch_id") {
+		object, err := kvstoreService.DescribeKVstoreInstance(d.Id())
+		if err != nil {
+			if NotFoundError(err) {
+				d.SetId("")
+				return nil
+			}
+			return WrapError(err)
+		}
```

Comment:
please use d.GetChange to get old and new value.

Reply:
done